cmd/recv.go

```diff
@@ -190,7 +190,7 @@ func recvAction(cmd *cobra.Command, args []string) {
 				}
 
 				tmpFile.Seek(0, io.SeekStart)
```

Comment:
Is this seek really necessary? We have just created the file, so seeking to the start should not be needed, as far as I know. The example does not use this and works just fine as is and I don't do it in wormhole-gui either.

Reply:
Does it work if you don't do this seek? We just wrote to this file handle so the file position indicator will be at the end of the file instead of the beginning.

Reply:
Yes. This works fine without seeking first. I am not doing it in the example and not in wormhole-gui, without any issues whatsoever.

Reply:
I re-tested with it removed in the cmd command code and recv works great without it.